web-app/js/portal/cart/DownloadEstimator.js

```diff
@@ -66,7 +66,7 @@ Portal.cart.DownloadEstimator = Ext.extend(Object, {
     _addDownloadEstimate: function(sizeEstimate, uuid, callback) {
 
         var htmlAddition = this._generateEstHtmlString(sizeEstimate);
-        var sizeDiv = Ext.get(this.getIdElementName(uuid));
+        var sizeDiv = Ext.select(this.getIdElementName(uuid));
```

Comment:
This would be the first time `Ext.get` is used in our code. Are you sure you need to change this?